generators/server/templates/src/main/java/package/config/SecurityConfiguration.java.ejs

```diff
@@ -232,7 +233,18 @@ public class SecurityConfiguration {
                 exceptionHanding.defaultAuthenticationEntryPointFor(
                     (HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) ->
                         response.setStatus(401),
-                    new AntPathRequestMatcher("/api/**")
+                        new OrRequestMatcher(
+  <%_ if (communicationSpringWebsocket) { _%>
+                            new AntPathRequestMatcher("/websocket/**"),
+  <%_ } _%>
+                            new AntPathRequestMatcher("/api/**"),
+                            /*
+                             * FormLogin is redirecting to "/error" instead of just passtrough.
+                             * Error started at https://github.com/jhipster/generator-jhipster/pull/20107.
+                             * Once the redirect cause is found this can be dropped in favor of the above "/api/**" entrypoint.
+                             */
+                            new AntPathRequestMatcher("/error")
+                        )
```

Comment:
This workaround seems like a bug. Should we enter something in the Spring Security project's issues?

Reply:
I am investigating

Reply:
It was hard, but is fixed.

Reply:
Excellent! I'll increase the bug bounty because of the difficulty.